ansible/vars/epoch/uat.yml

```diff
@@ -20,6 +20,7 @@ epoch_config:
       port: 3013
     internal:
       port: 3113
+    debug: true
```

Comment:
I see this is actually [this](https://github.com/aeternity/epoch/blob/master/apps/aeutils/priv/epoch_config_schema.json#L281) and not the tag `debug`? I am digging deeper

Reply:
Regarding this, I expect a behavior change introduced in aeternity/epoch#1901

Reply:
it is a bit confusing to understand the difference between [this](https://github.com/aeternity/epoch/blob/e8051ab20352a25727d15d3503c8640792258dca/apps/aeutils/priv/epoch_config_schema.json#L272) and [this](https://github.com/aeternity/epoch/blob/e8051ab20352a25727d15d3503c8640792258dca/apps/aeutils/priv/epoch_config_schema.json#L281) from the description. 

Maybe @fabiankrol can suggest what is the right way to enable the debug/internal endpoints?

Reply:
https://github.com/aeternity/epoch/pull/1901

Reply:
due to ongoing changes and other impacts, let's take out the `debug` change so the pr can be merged